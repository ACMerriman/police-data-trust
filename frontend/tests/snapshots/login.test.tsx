import * as React from "react"
import UserLogin from "../../pages/login"
import renderer from "react-test-renderer"

it("renders Login correctly", () => {
  const tree = renderer.create(<UserLogin />).toJSON()
  expect(tree).toMatchSnapshot()
})
