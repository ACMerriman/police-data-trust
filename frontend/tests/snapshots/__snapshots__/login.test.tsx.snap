// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Login correctly 1`] = `
<div
  className="Login"
>
  I AM LOGIN!!!!!!
</div>
`;
